app/helpers/schema_helper.py

```diff
@@ -35,7 +35,7 @@ def is_first_block_id_for_group(cls, survey_json, group_id, block_id):
 
     @staticmethod
     def get_last_block_id(survey_json):
-        return survey_json['groups'][0]['blocks'][-1]['id']
+        return survey_json['groups'][-1]['blocks'][-1]['id']
```

Comment:
Sure there is a good reason, why have we gone from the last block of the first group to the last block of the last group?